bundles/net.tourbook/src/net/tourbook/data/TourBike.java

```diff
@@ -53,9 +77,6 @@ public class TourBike implements Serializable {
     */
    private int               typeId;

```

Comment:
A type should not be an integer value, this is difficult to read, it should be an enum (string) value